### PR TITLE
Improved 3rd_party_voice_clients(experiment).md #50

### PR DESCRIPTION
**Pull request by albertopoljak**

### CODE REVIEW

File: docs/topics/3rd_party_voice_clients(experiment).md [hunk 1]
```diff
@@ -292,10 +292,10 @@ if (region is None):
...
-At cases you might need rich information about entities, but you only have their id available. Even tho most objects
-expose entity access too, like, they have `channel_id` attribute and a `channel` property, this may not be enough.
+In certain cases you might need rich information about entities, but you only have their ID available.
+Even tho most objects will expose entity access too, e.g. they have `channel_id` attribute and a `channel` property, this may not be enough.
 
-At these cases, use `USERS`, `GUILDS`, `CHANNELS` caches to access the entities.
+In these cases use `USERS`, `GUILDS`, `CHANNELS` caches to access the entities.
 
 ```py
 channel = CHANNESL.get(channel_id, None)
```
Comment (albertopoljak) [inline docs/topics/3rd_party_voice_clients(experiment).md:301]: I assume this is a typo? Waiting for verification.

**Comment by albertopoljak** [comment]
Also can I remove redundant parentheses, like these:
`if (voice_client is not None):`

File: docs/topics/3rd_party_voice_clients(experiment).md (hunk 1 above)
Comment (HuyaneMatsu) [reply docs/topics/3rd_party_voice_clients(experiment).md:301]: Indeed it is.

**Comment by HuyaneMatsu** [comment]
> Also can I remove redundant parentheses, like these: `if (voice_client is not None):`

If you feel like it.

**Comment by HuyaneMatsu** [comment]
👍🏻

**Pull request merged**